config.py

```diff
@@ -21,7 +21,6 @@
 param["report_sampling"] = 100
 # Custom timestamp
 timestamp = str(datetime.datetime.now().strftime("%Y%m%dT%H%M%S"))
```

Comment:
I moved the timestamp at the start of the config file, can be useful to have it handy to implement custom timestamps